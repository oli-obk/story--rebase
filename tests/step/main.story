//@ revisions: go_to_deeper go_to_void invalid_selection parser_error
//@[go_to_deeper,go_to_void] check-pass
entrance

## entrance
You enter a dark cave
out: gtfo
corridor: go deeper

## corridor
It's dark and your steps echo far ahead of your
deeper: walk on
entrance: return

## deeper
You enter a large cavern with glowing moss on the walls.
corridor: return
deeper2: walk on
crawlspace: explore a small crawlspace to your right
