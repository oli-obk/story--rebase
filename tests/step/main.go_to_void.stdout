You enter a dark cave
[gtfo]
[go deeper]
It's dark and your steps echo far ahead of your
[walk on]
[return]
You enter a dark cave
[gtfo]
[go deeper]
It's dark and your steps echo far ahead of your
[walk on]
[return]
You enter a large cavern with glowing moss on the walls.
[return]
[walk on]
[explore a small crawlspace to your right]
You fell off the end of the world
